pepys_import/core/store/data_store.py

```diff
@@ -6,20 +6,26 @@
 from getpass import getuser
 from importlib import import_module
 
+from alembic import command
+from alembic.config import Config
 from sqlalchemy import create_engine, inspect, or_
 from sqlalchemy.event import listen
 from sqlalchemy.exc import ArgumentError, OperationalError
 from sqlalchemy.orm import sessionmaker
 from sqlalchemy.sql import func, select
 
-from paths import PEPYS_IMPORT_DIRECTORY
+from paths import PEPYS_IMPORT_DIRECTORY, ROOT_DIRECTORY
 from pepys_import import __version__
 from pepys_import.core.formats import unit_registry
 from pepys_import.core.formats.location import Location
 from pepys_import.core.store import constants
 from pepys_import.resolvers.default_resolver import DefaultResolver
 from pepys_import.utils.branding_util import show_software_meta_info, show_welcome_banner
-from pepys_import.utils.data_store_utils import import_from_csv
+from pepys_import.utils.data_store_utils import (
+    create_spatialite_tables_for_postgres,
```

Comment:
Minor issue, but `spatialite` is the name of the spatial extension for SQLite, and `postgis` is the name of the spatial extension for postgres. Maybe just rename these to `create_spatial_tables_for_X`?

Reply:
Thanks for pointing out the typo. It's corrected:
https://github.com/debrief/pepys-import/pull/337/commits/77bfbb14b2fd4282d5498e75ff023337890ce8b2